tool/actions-plan-preview/github.go

```diff
@@ -55,10 +55,10 @@ type GraphQLClient interface {
 // parsePullRequestEvent uses the given environment variables
 // to parse and build githubEvent struct.
 // Currently, we support 2 kinds of event as below:
-// - PullRequestEvent
-//   https://pkg.go.dev/github.com/google/go-github/v36/github#PullRequestEvent
-// - IssueCommentEvent
-//   https://pkg.go.dev/github.com/google/go-github/v36/github#IssueCommentEvent
+//   - PullRequestEvent
+//     https://pkg.go.dev/github.com/google/go-github/v36/github#PullRequestEvent
+//   - IssueCommentEvent
+//     https://pkg.go.dev/github.com/google/go-github/v36/github#IssueCommentEvent
```

Comment:
Should keep this as is due to go convention 👀

Reply:
fix it in [this PR](https://github.com/pipe-cd/pipecd/pull/4648/commits/d0cb1a5deffb760baab87c916f6561958f4553e5)